### PR TITLE
Add release notes for 0.243.4

### PR DESCRIPTION
**Pull request by bhhari**
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.243.4.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+===============
+Release 0.243.4
+===============
+
+General Changes
```
Comment (mbasmanova) [inline presto-docs/src/main/sphinx/release/release-0.243.4.rst:5]: Shouldn't this be Hive Connector?

Comment (bhhari) [reply presto-docs/src/main/sphinx/release/release-0.243.4.rst:5]: will change to "Hive Changes"

**Review by mbasmanova — APPROVED** [review]

**Review by kaikalur — APPROVED** [review]

File: presto-docs/src/main/sphinx/release/release-0.243.rst [hunk 2]
```diff
@@ -8,6 +8,9 @@ Release 0.243
 .. warning::
     There is a bug that results in columns in the ``USING`` clause of a query to not be checked properly for column-level access control (:pr:`15333`).
 
+.. warning::
```
Comment (rschlussel) [inline presto-docs/src/main/sphinx/release/release-0.243.rst:11]: was this bug introduced in 0.243 or was it a long standing issue that we just discovered now?

Comment (bhhari) [reply presto-docs/src/main/sphinx/release/release-0.243.rst:11]: this was introduced in 0.243 https://github.com/prestodb/presto/pull/15304

**Review by rschlussel — APPROVED** [review]

**Pull request merged**